knitpy/knitpy.py

```diff
@@ -706,6 +706,12 @@ def render(self, filename, output=None):
         filename = os.path.abspath(filename)
         self.log.info("Converting %s..." % filename)
 
+        # no reason to continue past this point if we dont have
+        # a valid file path, so check it here and provide a clear
+        # failure message since this a common mistake
+        if not os.path.exists(filename):
```

Comment:
Can you have a look how that error message looks like when you are "just" in a wrong directory? As it it after the `abspath`, it has some mangling and will probably be funny for someone who has just used a filename?